bench/workgen/workgen.cpp

```diff
@@ -1851,6 +1871,18 @@ ThreadRunner::op_run(Operation *op)
         const std::shared_lock lock(*_icontext->_dyn_mutex);
         // For operations on random tables, if a table has been selected, decrement the
         // reference counter.
+        if(_icontext->_dyn_table_runtime[tint]._in_use == 0) {
+            std::cout << "_in_use should be greater than 0 but is not." << std::endl;
```

Comment:
You should use ```std:cerr``` instead of ```std:cout``` since at this point we will assert. Also since you have introduced this if condition here, we dont need the assert just after, we can just throw an exception:

```THROW("Table in use count went negative")```

Reply:
Sorry, I should have left a comment about those logs. I don't think I will leave them, I am chasing another bug. I am hitting the same assertion but this is not related to the missing base/mirror.

Reply:
Reverted in 119efc3bfdfc2c16cd6f42b97a79d4c114559715